### ISSUE TITLE
[Bug] Gadgetzan Water Survey item interaction area is too small

### ISSUE THREAD

**Issue by Arcitec** [azerothcore/azerothcore-wotlk#18871]
### Current Behaviour

Currently you have to stand in the absolute center of the water, as seen in this other ticket:

https://github.com/azerothcore/azerothcore-wotlk/issues/9984

It made the quest seem bugged to me but I persistently walked around and spammed the item until I found the tiny hotspot.

### Expected Blizzlike Behaviour

On official, you can do it anywhere at the water, even standing outside the water at any of the far edges...

https://www.wowhead.com/wotlk/quest=992/gadgetzan-water-survey#comments:id=2744560

### Steps to reproduce the problem

1. Quest.

### AC rev. hash/commit

AzerothCore rev. c9bdf4c0ef72 2024-05-06 21:45:46 +0200 (master branch) (Unix, RelWithDebInfo, Static)

### Operating system

Linux

**Comment by dewmguy** [comment]
i've seen this issue in TC as well

**Comment by heyitsbench** [comment]
Likely due to misplaced game object, checked a sniff near the area but it didn't show up. 😕

**Comment by dewmguy** [comment]
This is true. This has been a problem in all 1.12-3.35 emulators I've tested for as long as I can remember, including Mangos, TC and AC.
